static/app/views/replays/filters.tsx

```diff
@@ -17,25 +16,8 @@ const DEFAULT_QUERY = 'duration:>=5';
 
 function ReplaysFilters() {
   const {selection} = usePageFilters();
-  const location = useLocation();
+  const {pathname, query} = useLocation();
   const organization = useOrganization();
-  const didMount = useRef(false);
-
-  const {pathname, query} = location;
-
-  useEffect(() => {
-    if (!didMount.current && !query?.query) {
-      browserHistory.push({
-        pathname,
-        query: {
-          ...query,
-          cursor: undefined,
-          query: DEFAULT_QUERY,
-        },
-      });
-    }
-    didMount.current = true;
-  }, [pathname, query]);
```

Comment:
Looks like this was meant to inject the default query into the url on pageload. But we don't need any of that.

Instead the rules are simple:
- If there is no `?query=` in the url, then use the default query
- If the `?query=` param is set, use the value (either empty string, or some other string).

Reply:
Nice